default.nix

```diff
@@ -247,6 +247,8 @@ let
     maxLayers ? 1,
     # Deprecated: will be removed on v1
     contents ? null,
+    # Author, comment, command
```

Comment:
I don't think we can set `command` here.

Reply:
You're right - this comment should say `created_by`. Fixed.